# 车辆控制消息
float32 throttle     # 油门 (0-1)
float32 steer        # 转向 (-1左, 1右)
float32 brake        # 刹车 (0-1)
bool reverse         # 倒车
bool hand_brake      # 手刹